src/nt_users_tool/constants.py

```diff
@@ -0,0 +1,47 @@
+from lib2to3.pgen2.token import NAME
+
+
+BLANK = " "
+SLASH = "/"
+
+USERNAME = "user_name"
+FULL_NAME = "full_name"
+EXPIRATION_DATE = "expiration_date"
+DICT_INFO_LIST = [USERNAME, FULL_NAME, EXPIRATION_DATE]
+
+SHEET_EXPIRED = "expired_users"
+SHEET_EXPIRES_SOON = "expiring_soon_users"
+SHEET_ALL_USERS = "all_users"
+SHEETS_NAME_LIST = [SHEET_EXPIRED, SHEET_EXPIRES_SOON, SHEET_ALL_USERS]
+
+LAST_RELEVANT_ELEMENT_POSITION = 40
+
+NT_USER_COLUMN = "D"
+NAME_COLUMN = "E"
+EXPIRATION_DATE_COLUMN = "F"
+
+COLUMNS_LIST = [NT_USER_COLUMN, NAME_COLUMN, EXPIRATION_DATE_COLUMN]
+
+FAKE_NET_COMMAND = "The request will be processed at a domain controller for domain de.bosch.com\
```

Comment:
those fake strings, you might be able to move them in the unit tests part when you're doing them. They might be useful for mocking

Reply:
Still not moved with commit 1ea2511, will move them later on.

Reply:
Moved in ef87cb4.